Fix dnPhyloCTMC help page formatting

ms609 opened the pull request:
This corrects the halp page syntax to fix some formatting issues - see
https://revbayes.github.io/documentation/dnPhyloCTMC.html

Other fixes would be useful - the markdown does not seem to be rendering as expected, and I'm not sure how the 'example' section should be typeset so as to be displayed as code.

ms609 committed:
Fix formatting

This fixes some formatting issues - see
https://revbayes.github.io/documentation/dnPhyloCTMC.html

Other fixes would be useful - the markdown does not seem to be rendering as expected, and I'm not sure how the 'example' section should be typeset so as to be displayed as code.

davidcerny commented:
To get the example section to render properly, don't use backticks (` ``` `) but simply indent the code by 4 spaces. You can use (for example) the [`fnJC` help file](https://github.com/revbayes/revbayes/blob/master/help/md/fnJC.md?plain=1) as a template.

ms609 committed:
indent example

ms609 commented:
> Examples indented.

davidcerny commented on help/md/dnPhyloCTMC.md:
```diff
@@ -1,10 +1,12 @@
 ## description
+The parameters of a phylogenetic model – a tree topology with branch lengths, a substitution model that describes how observations evolve over the tree, etc. – collectively form a distribution called the _phylogenetic continuous-time Markov chain_.
+
 dnPhyloCTMC gives the probability distribution of tip labels on a phylogenetic tree given an phylogenetic continuous-time Markov chain model.
```
While we're at it, we might want to change line 10 as well – I find it a little confusing. `dnPhyloCTMC` gives the probability distribution of _character state vectors at the tips_ of a phylogenetic tree. I guess you could argue that tips are labeled by their character state vectors, but normally the phrase "tip label" would refer to the taxon name. Also, there is a typo: "an phylogenetic" -> "a phylogenetic".

ms609 replied:
I agree – Updated per your suggested wording

davidcerny committed:
Merge branch 'development' into ms609-dnPhyloCTMC-format

ms609 committed:
Clarify description of dnPhyloCTMC functionality

Update wording per @davidcerny comments

Also replace em-dashes with markdown encoding - might fix Windows mangling of UTF-8 characters?

ms609 committed:
an→a
